igvc/package.xml

```diff
@@ -45,23 +47,22 @@
   <build_depend>pluginlib</build_depend>
   <build_depend>class_loader</build_depend>
   <build_depend>image_geometry</build_depend>
+
+  <!-- Required -->
   <run_depend>roscpp</run_depend>
   <run_depend>std_msgs</run_depend>
   <run_depend>image_transport</run_depend>
   <run_depend>joystick_drivers</run_depend>
   <run_depend>cv_bridge</run_depend>
   <run_depend>pcl_conversions</run_depend>
-  <run_depend>laser_geometry</run_depend>
-  <run_depend>stereo_image_proc</run_depend>
-  <run_depend>sick_tim</run_depend>
-  <run_depend>camera_info_manager</run_depend>
   <run_depend>rviz</run_depend>
-  <run_depend>class_loader</run_depend>
-  <run_depend>pluginlib</run_depend>
-  <run_depend>class_loader</run_depend>
```

Comment:
How was this ruled out?

Reply:
I searched the code and to see if any of them were used.